static/app/components/events/interfaces/breadcrumbs/breadcrumb/data/linkedEvent.tsx

```diff
@@ -96,10 +98,9 @@ function LinkedEvent({orgSlug, eventId, route, router}: Props) {
         return;
       }
 
-      addErrorMessage(
-        t('An error occurred while fetching the data of the breadcrumb event link')
-      );
-      Sentry.captureException(error);
+      addErrorMessage(errorMessage);
+      handleXhrErrorResponse(errorMessage)(error);
```

Comment:
Should this be treated as the same error as `fetchEventById`? This seems to only be called after `fetchEventById` succeeds, so it seems like if this API call fails, it may be because of a different reason.

Reply:
the user wouldn't know that we first need to fetch data from an issue that we use to fetch data from an event, but I've updated the second `handleXhrErrorResponse` message so we can differentiate these two errors in Sentry